### ISSUE TITLE
Add option to back up paths from file

### ISSUE THREAD

**Issue by IronCraftMan** [borgbackup/borg#8103]
## Have you checked borgbackup docs, FAQ, and open GitHub issues?

Yes

## Is this a BUG / ISSUE report or a QUESTION?

Feature

#### Your borg version (borg -V).
1.2.7
#### Operating system (distribution) and version.
macOS
#### Hardware / network configuration, and filesystems used.
N/A
#### How much data is handled by borg?
N/A
#### Full borg commandline that lead to the problem (leave away excludes and passwords)
`cat file.txt | borg create --paths-from-stdin ...`

## Describe the problem you're observing.

I have a text file with a list of paths to back up. I tried using `--paths-from-stdin`, only to discover that it only backs up the exact folder, without recursion. My file contains a list of folders that all need to be backed up, and I don't think it's feasible to recreate it before every backup before first scanning through all the directories. I didn't find any other way I can send a text file full of paths to borg to back up.

Ideally, something like restic's `--files-from` option would work, where it just pulls a newline-separated list of files.

**Comment by infectormp** [comment]
Try `--patterns-from`
https://borgbackup.readthedocs.io/en/stable/usage/help.html#borg-help-patterns

**Comment by ThomasWaldmann** [comment]
@infectormp Yeah, guess one could feed multiple `R /recursion/root`-style lines via that. If there are only a "few", just giving them as cli args would also work.

@IronCraftMan about `--paths-from-*`:

borg is rather strict and clean here:
- either borg is in control (normal operation, without that option): does the recursion, finds files, processes excludes/includes
- or the user takes full control (with that option) - but then they also need to do **all** of that and feed a list of all files that shall be in the archive.

**Comment by jdchristensen** [comment]
> If there are only a "few", just giving the paths as cli args would also work.

E.g. one could do:  ``borg create repo::archive `cat file.txt` `` (if they fit on the command line).
